donate/thunderbird/templates/fragments/donate_form_disclaimer.html

```diff
@@ -9,7 +9,7 @@
 {% endblock %}
 
 {% block faq_contact %}
-    {% blocktrans with faq_url='/faq' email='thunderbird-donate@mozilla.org' trimmed %}
+    {% blocktrans with faq_url='/faq' email='give@thunderbird.net' trimmed %}
```

Comment:
Is this a real email address? @stephaniemcv?

Reply:
Those are both real email addresses.